Update devices.js

Vodochnik opened the pull request:
Add temperature expose for Aqara RTCGQ11LM body movement and illuminance sensor

asgothian reviewed (changes requested):
Please test if the states generated if line 725 is removed from the code are identical in name and function to the states generated with the line present.

Note that additional states appearing would be ok, states missing would not be ok.

If the test is successful (all states remain and remain active), i would propose to remove line 725 in order to move towards using herdsman exposes as main method for generating states.

Vodochnik commented:
![grafik](https://user-images.githubusercontent.com/4112750/122639954-17d3ec80-d0fd-11eb-9460-1def8c67bd71.png)
Tried to delete line 725 as you suggested, old motion sensor works, all states remain but no_motion has no function now.

After this I have added a brand new sensor (same model) and no_motion and occupancy_timeout were not created. Some users can rely on them, so I don't think it's a good idea to delete the line, for me I have reverted the deletion.

asgothian commented:
Thanks for the test. Due to the loss of functions, if the change is made impushed the change through.

A.
